### PR TITLE
Create release

### PR DESCRIPTION
**Pull request by blagoev**
creates a draft release with all artifacts

### CODE REVIEW

File: .github/workflows/realm-dart-linux.yml [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 on:
   push:
-    branches: [ master ]
+    branches:
     # Do not run if change is in these paths
     paths-ignore:
       - 'flutter/**'
       - 'generator/**'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+**' # matches tags like number(s).number(s).number(s)(any) for ex: 1.0.0 and also 1.0.0+beta
```
Comment (nielsenko) [inline .github/workflows/realm-dart-linux.yml:11]:
```suggestion
      - '[0-9]+.[0-9]+.[0-9]+.*' # matches tags like number(s).number(s).number(s)(any) for ex: 1.0.0 and also 1.0.0+beta
```

**Review by nielsenko — CHANGES_REQUESTED** [review]

File: .github/workflows/realm-dart-windows.yml [hunk 2]
```diff
@@ -39,4 +52,27 @@ jobs:
...
+
+      - name: Release artifacts
+        if: ${{ success() && github.event_name == 'push' && github.ref_type == 'tag' }}
+        uses: ncipollo/release-action@v1
+        with:
+          allowUpdates:  true
+          artifactErrorsFailBuild: true
+          draft: true
+          omitNameDuringUpdate: true
+          prerelease: false
```
[the remark this answers is not in the document]
Comment (blagoev) [reply .github/workflows/realm-dart-windows.yml:73]: Lets do only Draft releases for now.

File: .github/workflows/realm-flutter-android.yml [hunk 3]
```diff
@@ -2,11 +2,13 @@ name: Realm Flutter for Android
...
 on:
   push:
-    branches: [ master ]
+    branches:
     # paths:
     #   - 'flutter/**'
     #   - 'src/**'
     #   - 'lib/**'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+**' # matches tags like number(s).number(s).number(s)(any) for ex: 1.0.0 and also 1.0.0+beta
```
[the remark this answers is not in the document]
Comment (blagoev) [reply .github/workflows/realm-flutter-android.yml:11]: this `**` mean any number of characters following.

File: .github/workflows/realm-flutter-windows.yml [hunk 4]
```diff
@@ -2,11 +2,13 @@ name: Realm Flutter for Windows
...
 on:
   push:
-    branches: [ master ]
+    branches:
     # Do not run if change is in these paths
     paths-ignore:
       - 'flutter/**'
       - 'generator/**'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+**' # matches tags like number(s).number(s).number(s)(any) for ex: 1.0.0 and also 1.0.0+beta
```
[the remark this answers is not in the document]
Comment (blagoev) [reply .github/workflows/realm-flutter-windows.yml:11]: as above

File: .github/workflows/realm-flutter-ios.yml [hunk 5]
```diff
@@ -2,11 +2,13 @@ name: Realm Flutter for iOS
...
 on:
   push:
-    branches: [ master ]
+    branches:
     # paths:
     #   - 'flutter/**'
     #   - 'src/**'
     #   - 'lib/**'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+**' # matches tags like number(s).number(s).number(s)(any) for ex: 1.0.0 and also 1.0.0+beta
```
[the remark this answers is not in the document]
Comment (blagoev) [reply .github/workflows/realm-flutter-ios.yml:11]: as above

File: .github/workflows/realm-flutter-macos.yml [hunk 6]
```diff
@@ -2,11 +2,13 @@ name: Realm Flutter for MacOS
...
 on:
   push:
-    branches: [ master ]
+    branches:
     # Do not run if change is in these paths
     paths-ignore:
       - 'flutter/**'
       - 'generator/**'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+**' # matches tags like number(s).number(s).number(s)(any) for ex: 1.0.0 and also 1.0.0+beta
```
[the remark this answers is not in the document]
Comment (blagoev) [reply .github/workflows/realm-flutter-macos.yml:11]: as above

File: .github/workflows/realm-dart-windows.yml [hunk 7]
```diff
@@ -2,11 +2,13 @@ name: Realm Dart for Windows
...
 on:
   push:
-    branches: [ master ]
+    branches:
     # Do not run if change is in these paths
     paths-ignore:
       - 'flutter/**'
       - 'generator/**'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+**' # matches tags like number(s).number(s).number(s)(any) for ex: 1.0.0 and also 1.0.0+beta
```
[the remark this answers is not in the document]
Comment (blagoev) [reply .github/workflows/realm-dart-windows.yml:11]: as above

File: .github/workflows/realm-dart-linux.yml (hunk 1 above)
Comment (blagoev) [reply .github/workflows/realm-dart-linux.yml:11]: as above

File: .github/workflows/realm-flutter-android.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (nielsenko) [reply .github/workflows/realm-flutter-android.yml:11]: No `**` is an invalid regex pattern - '.*' means any number of characters '.' = any, '*' = 0 or more

[the remark this answers is not in the document]
Comment (nielsenko) [reply .github/workflows/realm-flutter-android.yml:11]: You cannot match globs and regex like that .. to my knowledge

[the remark this answers is not in the document]
Comment (blagoev) [reply .github/workflows/realm-flutter-android.yml:11]: This syntax is from here https://docs.github.com/en/enterprise-server@3.0/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
> ** Matches zero or more of any character.

[the remark this answers is not in the document]
Comment (nielsenko) [reply .github/workflows/realm-flutter-android.yml:11]: Arhh.. this is not a classic regex.. it just kind of look like one: https://docs.github.com/en/enterprise-server@3.0/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

**Review by nielsenko — APPROVED** [review]

**Review by desistefanova — APPROVED** [review]
.github/workflows/realm-dart-linux.yml has two more white spaces at the beginning "name: Realm   Dart for Linux"

File: .github/workflows/realm-dart-linux.yml [hunk 8]
```diff
@@ -1,12 +1,14 @@
-name: Realm Dart for Linux
+name: Realm   Dart for Linux
```
Comment (desistefanova) [inline .github/workflows/realm-dart-linux.yml:1]: There are two more white spaces "name: Realm   Dart for Linux"

Comment (blagoev) [inline .github/workflows/realm-dart-linux.yml:1]:
```suggestion
name: Realm Dart for Linux
```

Comment (blagoev) [reply .github/workflows/realm-dart-linux.yml]: fixed

**Pull request merged**